lib/msal-angular/src/msal.broadcast.config.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export type MsalBroadcastConfiguration = {
+    replayPastEvents: number
```

Comment:
The name sounds like it is a boolean, I'd go for something like `eventsToReplay`

Reply:
Makes sense. Changed.